keyboards/ryanskidmore/rskeys100/matrix.c

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2014 Ralf Schmitt <ralf@bunkertor.net>
+Modified by Ryan Skidmore <rskeys@ryanskidmore.co.uk> (@ryanskidmore)
+to support the rskeys100.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <avr/io.h>
+#include <util/delay.h>
+#include "print.h"
+#include "debug.h"
+#include "util.h"
+#include "led.h"
+#include "config.h"
+#include "matrix.h"
+#include "quantum.h"
+#include "debounce.h"
+
+#ifndef DEBOUNCE
+#    define DEBOUNCE 5
+#endif
```

Comment:
Some of these may not be needed.

Reply:
Good spot - I've removed the unused ones now.